1-mandatory/1-writers.js

```diff
@@ -51,7 +51,9 @@ Exercise 1:
   "Hi, my name is {firstName} {lastName}. I am {age} years old, and work as a {occupation}."
 */
 function logAllWriters() {
-  // write your code to log all writers here
+  for (author of writers) { 
+    console.log (`Hi, my name is ${author.firstName} ${author.lastName}. I am ${author.age} years old, and work as a ${author.occupation}.`)
+  }
 }
```

Comment:
Comment: Well done.

Consider: Here is another way d it. function logAllWriters() {
  // write your code to log all writers here
  for (let i = 0; i < writers.length; i++) {
    const writer = writers[i];
    console.log(`Hi, my name is ${writer.firstName} ${writer.lastName}. I am ${writer.age} years old, and work as a ${writer.occupation}.`);
  }
}